MicrosoftFluentUI.podspec

```diff
@@ -1,11 +1,11 @@
 Pod::Spec.new do |s|
   s.name             = 'MicrosoftFluentUI'
-  s.version          = '0.4.8'
+  s.version          = '0.5.102'
   s.summary          = 'Fluent UI is a set of reusable UI controls and tools'
   s.homepage         = "https://www.microsoft.com/design/fluent/#/"
   s.license          = { :type => 'MIT', :file => 'LICENSE' }
   s.author           = { "Microsoft" => "fluentuinativeowners@microsoft.com"}
-  s.source       = { :git => "https://github.com/microsoft/fluentui-apple.git", :tag => "#{s.version}" }
+  s.source       = { :git => "https://github.com/microsoft/fluentui-apple.git", :branch => 'mischreiber/main-tokens', :tag => "#{s.version}_main_tokens" }
```

Comment:
this is temporary not not being checked in? or it hsould be eventually just main_tokens?

Reply:
Good catch, should be temporary, but was needed to generate a test CocoaPod for our downstream dependencies.